### PR TITLE
Clean up design index page

### PR DESCRIPTION
**Pull request by Badlapje**
Ensure display on mobile looks good.  
Clean up display on desktop so that the title / footer are aligned along with the body

### CODE REVIEW

**Comment by MKodde** [comment]
The changes look good +1 for the stylesheet cleanup you added to this PR!

I noticed some regression in your template/stylesheet changes. See the screenshots for a before/after. Maybe this is good enough, but maybe you'd like to address them?


**Screenshot from engine.surfconext.nl (current)**
Screenshot from 2021-01-05 08-48-54

**From your branch**
Screenshot_2021-01-05 OpenConext - Metadata Certificates

What caught my eye:

1. Mobile lookin' good!
2. We lost the drop shadow in your branch
3. Margin above h1 `IdP Certificate and Metadata` should be increased on top? Or maybe padding on the `hr` right above should be increased?
4. The header font-size is decreased? Or is this due to the increased header size? It seems more compact on the 'before' shot

File: theme/base/stylesheets/front-page/modules/container.scss [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    .strong {
+        font-weight: bold;
+    }
+
+    .box {
+        background: #fff;
+        border-radius: $br;
+        box-shadow: 0 1px $br rgba(black,.2);
+    }
```
Comment (MKodde) [inline theme/base/stylesheets/front-page/modules/container.scss:81]: Think this class should be applied on the content 'box'? For theyn dwelleth not in the darkness of the boxe shadow.

File: theme/base/stylesheets/front-page/modules/header.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            }
+        }
+
+        .burger {
+            display: block;
+
+            .meta {
+                display: none;
+            }
+        }
```
Comment (MKodde) [inline theme/base/stylesheets/front-page/modules/header.scss:75]: Yum! Reviewing this page, I find the burger behaves rather strange. Whats the point in having a burger (which hints on opening a menu?) to jump to the footer? Not something to address here. But something to review when we restyle this page to the skeune theme?

Comment (Badlapje) [reply theme/base/stylesheets/front-page/modules/header.scss:75]: I fully agree 😄   I was wondering about that as well.  I'll ask Arnout about it.

File: theme/base/stylesheets/front-page/modules/container.scss (hunk 1 above)
Comment (Badlapje) [reply theme/base/stylesheets/front-page/modules/container.scss:81]: Right thou arest!

File: theme/base/stylesheets/front-page/modules/header.scss (hunk 2 above)
Comment (Badlapje) [reply theme/base/stylesheets/front-page/modules/header.scss:75]: and removed it is 🚀

**Comment by Badlapje** [comment]
Wel spotted on all counts!  I think i addressed them all in this rework.

**Review by MKodde — COMMENTED** [review]
Nice work, looks good! I see the CI checks are not passing at this moment. Try to fix those please.

And maybe consider moving the unused burger leftover from the base SASS into the openconext theme where it might be used. 

```
OpenConext-engineblock/theme$ grep -ir burger
base/stylesheets/error-page/modules/_error-container.sass:    .burger
```

**Pull request merged**